src/inc/corprof.idl

```diff
@@ -629,11 +629,14 @@ typedef enum
 
     COR_PRF_HIGH_BASIC_GC                           = 0x00000010,
 
+    COR_PRF_HIGH_MEDIUM_GC                          = 0x00000020,
```

Comment:
How about we change this to a more explicit name? I'm open to other suggestions, here is one: COR_PRF_HIGH_MONITOR_GC_MOVED_OBJECTS.

I also think it would be good to resume the tradition we had in the past (see higher up in the idl) of adding a comment above the enum value which describes the set of callbacks the flag enables.